hnn_core/tests/test_gui.py

```diff
@@ -492,32 +479,25 @@ def test_gui_adaptive_spectrogram():
     plt.close('all')
 
 
-@pytest.fixture
-def setup_gui():
-    gui = HNNGUI()
-    gui.compose()
-    gui.params['N_pyr_x'] = 3
-    gui.params['N_pyr_y'] = 3
-    gui.run_button.click()
-    return gui
-
-
-@pytest.mark.parametrize("viz_type", ["layer2 dipole", "layer5 dipole",
-                                      "spikes", "PSD", "network"])
-def test_gui_visualization(setup_gui, viz_type):
+def test_gui_visualization(setup_gui):
     """Test visualization functionality in the HNNGUI."""
     gui = setup_gui
+    gui.run_button.click()
```

Comment:
Yep! That's correct. The run was in the fixture previously. I moved it out because the fixture now goes to different tests and those tests might need to make some configuration adjustments before running the simulation.